java/compaction/compaction-core/src/main/java/sleeper/compaction/core/job/creation/CreateCompactionJobs.java

```diff
@@ -220,7 +220,7 @@ private void batchCreateJobs(AssignJobIdToFiles assignJobIdToFiles, TablePropert
         assignJobIdToFiles.assignJobIds(compactionJobs.stream()
                 .map(CompactionJob::createAssignJobIdRequest)
                 .collect(Collectors.toList()), tableStatus);
-        LOGGER.debug("Created pending compaction jobs batch for table {} in data bucket at: {}", tableStatus, request.getBatchKey());
+        LOGGER.debug("Created pending compaction jobs batch for table {} of size {}, in data bucket at: {}", tableStatus, compactionJobs.size(), request.getBatchKey());
```

Comment:
Looks like this was committed to the wrong branch accidentally. I'll just merge it in a separate PR:
- https://github.com/gchq/sleeper/issues/3863